src/main/charts/bitbucket/templates/statefulset-mesh.yaml

```diff
@@ -36,6 +36,17 @@ spec:
         {{- include "bitbucket.additionalHosts" . | nindent 8 }}
       initContainers:
         {{- include "bitbucket.mesh.additionalInitContainers" . | nindent 8 }}
+        {{- if and .Values.monitoring.exposeJmxMetrics .Values.monitoring.fetchJmxExporterJar }}
+        - name: fetch-jmx-exporter
+          image: {{ .Values.monitoring.jmxExporterImageRepo}}:{{ .Values.monitoring.jmxExporterImageTag}}
+          command: ["cp"]
+          args: ["/opt/bitnami/jmx-exporter/jmx_prometheus_javaagent.jar", "{{ .Values.bitbucket.mesh.volume.mountPath }}"]
```

Comment:
Is it because of the mesh in the path that we can't use the `common` package?

Reply:
Mesh does not mount shared-home, so the jar is downloaded to local-home instead. Perhaps it makes sense to add a condition in common helpers but 1.1.0 is released already and JMX for mesh was added after 1.1.0 was released.

Reply:
This is fine for now. We can potentially gather more additions to the next version of `common` package and then fix it later.